resources/inputmaps/keyboard.json

```diff
@@ -52,12 +52,14 @@
     "Ctrl\\+Q": "host:close",
     "Ctrl\\+Shift\\+R": "host:reload",
     "Ctrl\\+Shift\\+D": "host:toggleDebug",
+    "Ctrl\\+Shift\\+A": "host:alwaysOnTop",
```

Comment:
I think you should be able to use `host:cycle_setting alwaysOnTop` here that way you won't have to add a new host command.